composefiles/swarm-uniflow.yml

```diff
@@ -90,7 +90,10 @@ services:
       replicas: 1
 
   dashboard:
-    image: frinx/frinx-dashboard:1.0.1
```

Comment:
Add health check pls

Reply:
@jrozsa - what was the healthcheck until now or how should a health-check ideally look like?

Reply:
I merged the one meant for old dashboard yesterday, it should be in master.
 Just a get request and check response code is ok.

Reply:
@jrozsa ok - I added the healthcheck and rebased on latest master